python/tests/test_memory.py

```
@@ -0,0 +1,193 @@
+import pytest
+import copy
```

Ruff (0.8.2)

`copy` imported but unused

Remove unused import: `copy`

(F401)